builder/pipeline.h

```diff
@@ -192,28 +192,26 @@ class func {
   static func make(callable_wrapper<const In1, Out1> impl, input in1, output out1) {
     symbol_id in1_sym = in1.sym();
     symbol_id out1_sym = out1.sym();
-    return func(
-        [=, impl = std::move(impl)](eval_context& ctx) -> index_t {
-          const raw_buffer* in1_buf = ctx.lookup_buffer(in1_sym);
-          const raw_buffer* out1_buf = ctx.lookup_buffer(out1_sym);
-          return impl(in1_buf->cast<const In1>(), out1_buf->cast<Out1>());
-        },
-        {std::move(in1)}, {std::move(out1)});
+    const auto callable_wrapper = [=, impl = std::move(impl)](eval_context& ctx) -> index_t {
+      const raw_buffer* in1_buf = ctx.lookup_buffer(in1_sym);
+      const raw_buffer* out1_buf = ctx.lookup_buffer(out1_sym);
+      return impl(in1_buf->cast<const In1>(), out1_buf->cast<Out1>());
+    };
+    return func({callable_wrapper, "callable_wrapper"}, {std::move(in1)}, {std::move(out1)});
```

Comment:
Before merging this PR, we need to figure out what it's going to look like to pass a useful name here. It's not really an upgrade to see every call is a call to `"callable_wrapper"` everywhere in a printed IR statement instead of `"<fn>"`. I get that we can just pass an extra string here and do that everywhere, but I didn't do that for a reason, I don't like the extra boilerplate it adds to API usage. I feel the API is already very heavy on boilerplate.